eva-server/src/test/java/uk/ac/ebi/eva/server/ws/VariantWSServerV2IntegrationTest.java

```diff
@@ -210,4 +210,31 @@ public void sourceEntriesEndpointTestForError() throws URISyntaxException {
         assertEquals("Please specify either both annotation VEP version and annotation VEP cache version, " +
                 "or neither", testForErrorHelper(url));
     }
+
+    @Test
+    public void rootTestForAdditions() throws URISyntaxException {
+        String url = "/v2/variants/13:32889711:T:?species=mmusculus&assembly=grcm38";
+        VariantWithSamplesAndAnnotation variantWithSamplesAndAnnotations = restTemplate.exchange(
+                url, HttpMethod.GET, null,
+                new ParameterizedTypeReference<VariantWithSamplesAndAnnotation>() {
+                }).getBody();
+        assertEquals("13", variantWithSamplesAndAnnotations.getChromosome());
+        assertEquals("T", variantWithSamplesAndAnnotations.getReference());
+        assertNull(variantWithSamplesAndAnnotations.getAlternate());
```

Comment:
The other bugfix (besides the bugfix about `split`) we were talking about is exactly this. We would expect this to be an empty string, not null. The same for the reference allele in the insertion.